### PR TITLE
HTML - Web - Page - Addy Abid

### PR DESCRIPTION
**Pull request by AddyAbid**
Screen Shot 2021-08-17 at 5 39 26 PM

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint ducimus reprehenderit voluptas beatae non dolorum facere at molestiae similique quod cumque saepe officiis ipsa sit mollitia, nulla perferendis corrupti labore?
+  </p>
+
+  <table>
+   <tr>
+     <th></th>
+     <th>Basic Plan</th>
+     <th>Deluxe Plan</th>
+     <th>Jelly Giant Plan</th>
+   </tr>
```
Comment (codingcodymiller) [inline html-web-page/index.html:34]: This should be in a `thead` 👀

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    <th></th>
+    <th></th>
+    <th>&check;</th>
+   </tr>
+   <tr>
+     <td>Monthly Pricing</td>
+     <th>$30</th>
+     <th>$50</th>
+     <th>$60</th>
+   </tr>
```
Comment (codingcodymiller) [inline html-web-page/index.html:64]: This should be in a `tbody`. Remember to go back and reference previous exercises if you're unsure about how to do something, you just got this feedback on the last exercise as well.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here! A couple notes for you:

**Comment by AddyAbid** [comment]
Screen Shot 2021-08-17 at 5 49 53 PM

File: html-web-page/index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        <th>Deluxe Plan</th>
+        <th>Jelly Giant Plan</th>
+      </tr>
+    </thead>
+    <tbody>
+   <tr>
+     <td>Three new curated jellies per month</td>
+     <th>&check;</th>
+     <th>&check;</th>
+     <th>&check;</th>
```
Comment (codingcodymiller) [inline html-web-page/index.html:42]: These checkmarks are meant to be "table data", so they should be `td` elements. Make sure you change the others in your document as well 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by AddyAbid** [comment]
Screen Shot 2021-08-17 at 5 59 33 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**